ipie/addons/ephqmc/trial_wavefunction/toyozawa.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import scipy.linalg
+
+from ipie.addons.eph.trial_wavefunction.coherent_state import CoherentStateTrial
+from ipie.utils.backend import arraylib as xp
+from ipie.addons.eph.trial_wavefunction.variational.toyozawa_variational import circ_perm
+
+class ToyozawaTrial(CoherentStateTrial):
+    """"""
```

Comment:
Great place for docstring with math + references to source (i.e. journal). See: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages for how to write links in rst.